tests/acceptance/change.spec.js

```diff
@@ -0,0 +1,319 @@
+'use strict';
+
+var expect = require('chai').expect;
+var path = require('path');
+var change = require('../../addon/ng2/utilities/change');
+var mockFs = require('mock-fs');
+var existsSync = require('exists-sync');
+var fsPromise = require('../../addon/ng2/utilities/fs-promise');
+
+describe('Change', () => {
+  var sourcePath;
+
+  describe('Invalid Path', () => {
+
+    beforeEach(() => {
+      var mockDrive = {
```

Comment:
One other point to apply throughout: prefer `let` to `var`.

https://www.typescriptlang.org/docs/handbook/variable-declarations.html